for-devs.md

```diff
@@ -0,0 +1,83 @@
+## Developer Information
+
+This file is intended for developers. It provides details on the project's structure and instructions on how to work
+with it.
+
+### 1. Fixed Node Version Warning
+
+The project requires Node.js version 20 and is not compatible with later versions. You can
```

Comment:
We support newer versions as of pretty recently, see https://github.com/prosopo/captcha/pull/1540

Reply:
Great thing! Happy to remove it